shopfloor/services/location_content_transfer.py

```diff
@@ -855,17 +855,25 @@ def stock_out_package(self, location_id, package_level_id):
             # We need to set qty_done at 0 because otherwise
             # the move_line will not be deleted
             package_move.move_line_ids.write({"qty_done": 0})
-            package_move._do_unreserve()
-            package_move._recompute_state()
+            package = package_level.package_id
+            if (
+                self.is_allow_move_create()
+                and self.env.user == package_move.picking_id.create_uid
+            ):
+                # Owned by the user deleting the move
+                package_move._do_unreserve()
+                package_move._recompute_state()
+                package_move._action_cancel()
```

Comment:
Looks good to call `_action_cancel` before before creating the stock issue.
`_action_cancel` is already doing the `_do_unreserve`
```suggestion
            move._action_cancel()
```

Also update docstring